rtlib/src/LandmarkDetector.cpp

```diff
@@ -0,0 +1,61 @@
+#include "rt/LandmarkDetector.hpp"
+
+#include <iostream>
+#include <opencv2/features2d.hpp>
+#include <opencv2/highgui.hpp>
+
+using namespace rt;
+
+std::vector<LandmarkDetector::LandmarkPair> LandmarkDetector::compute(int num_matches) {
+    auto brisk = cv::BRISK::create();
+
+    auto matcher = cv::DescriptorMatcher::create("BruteForce");
+    // Match between fixedImg and movingImg
+    std::vector<cv::DMatch> matches;
+    // keypoint  for fixedImg and movingImg
+    std::vector<cv::KeyPoint> keyImg1, keyImg2;
+    // Descriptor for fixedImg and movingImg
+    cv::Mat descImg1, descImg2;
+    std::vector<LandmarkDetector::LandmarkPair> matched_points;
+
+    try {
```

Comment:
Get rid of this try catch block, I think. The application should do the try catch, not the library routine.